benchmark/client/main.go

```diff
@@ -16,6 +16,27 @@
  *
  */
 
+/*
+Package main provides a client used for benchmarking.  Before running the
+client, the user would need to launch the grpc server before running the
+client.
+
+To start the server before running the client, you can run something similar
+to the command below:
+
+go run benchmark/server/main.go -test_name=grpc_test
```

Comment:
Instead of duplicating this command here, can you provide a pointer to the other file?

Reply:
@zelahi can you address this one as well?

Reply:
apologies, I'll be addressing this in another commit